backend/src/main/java/atlantbh/restaurants/exceptions/RepositoryException.java

```diff
@@ -0,0 +1,14 @@
+package atlantbh.restaurants.exceptions;
+
+import org.hibernate.service.spi.ServiceException;
+
+public class RepositoryException extends ServiceException {
```

Comment:
public class RepositoryException extends Exception {

	public RepositoryException(String errorMessage) {
		super(errorMessage);
	}

	public RepositoryException(String errorMessage, Throwable cause) {
		super(errorMessage, cause);
	}

	@Override
	public String toString() {
		return getMessage();
	}
}

Reply:
Fixed in 8c841a20e489a31c66d177c8682339f9d4f19de0